example.yml

```diff
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://raw.githubusercontent.com/jskherman/cv.typ/main/cv.typ.schema.json
+# yaml-language-server: $schema=./cv.typ.schema.json
```

Comment:
Could we also return the original URL here please? The value here is relative to the root and, thus, won't work with submodules.